### PR TITLE
Feat: Add type into the GraphQL Location schema

### PR DESCRIPTION
**Pull request by FilipMessa**
Added type field into the Location schema.

### CODE REVIEW

File: apps/graphql/src/apps/location/types/outputs/Location.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ export default new GraphQLObjectType({
     coordinates: {
       type: GraphQLCoordinate,
     },
+    type: {
+      type: GraphQLString,
```
Comment (tbergq) [inline apps/graphql/src/apps/location/types/outputs/Location.js]: Maybe it should be an enum?

Comment (FilipMessa) [reply apps/graphql/src/apps/location/types/outputs/Location.js]: I was thinking about that, but I can't find any documentation about all possible values from TequilaAPI. But I asked for that so we will see.

File: apps/graphql/src/apps/location/types/enums/LocationType.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import { GraphQLEnumType } from 'graphql';
+
+export default new GraphQLEnumType({
+  name: 'LocationType',
+  values: {
+    station: {
```
Comment (tbergq) [inline apps/graphql/src/apps/location/types/enums/LocationType.js]: Values should be UPPER_CASED

File: apps/graphql/src/apps/location/dataloaders/Locations.js [hunk 3]
```diff
@@ -17,12 +17,34 @@ export type LocationInput =
       +code: string,
     |};
 
+function sanitizeLocationType({ type }: ApiLocation) {
+  if (type) {
+    return type.replace(/_(\w)/g, m => m[1].toUpperCase());
```
Comment (tbergq) [inline apps/graphql/src/apps/location/dataloaders/Locations.js]: could you add a comment as to why this is necessary?

File: apps/graphql/src/apps/location/types/enums/LocationType.js (hunk 2 above)
Comment (FilipMessa) [reply apps/graphql/src/apps/location/types/enums/LocationType.js]: So this it is the convention for Enums ?

Comment (tbergq) [reply apps/graphql/src/apps/location/types/enums/LocationType.js]: seems like it yeah, all enums in `incubator/universe` follow this convention, and all enums we have in margarita so far.

File: apps/graphql/src/apps/location/dataloaders/Locations.js (hunk 3 above)
Comment (FilipMessa) [reply apps/graphql/src/apps/location/dataloaders/Locations.js]: done

Comment (tbergq) [reply apps/graphql/src/apps/location/dataloaders/Locations.js]: Ok, now that I know why we does this, it seems unnecessary to me. 

```
BUS_STATION: {
      value: 'bus_station',
},
```

will resolve this to the enum anyway.

Comment (FilipMessa) [reply apps/graphql/src/apps/location/dataloaders/Locations.js]: 🚀

**Pull request merged**